packages/mco/constants/disaster-recovery.ts

```diff
@@ -9,6 +9,8 @@ export const DRPC_NAMESPACE_ANNOTATION =
 // DRPC annotations
 export const LAST_APP_DEPLOYMENT_CLUSTER_ANNOTATION =
   'drplacementcontrol.ramendr.openshift.io/last-app-deployment-cluster';
+export const APP_NAMESPACE_ANNOTATION =
```

Comment:
Name should be intuitive enough to know which app type it belongs to...
```suggestion
export const MANAGED_APP_NS_ANNOTATION =
```

Reply:
The idea is to introduce this annotation for subscription as well as Appset, Due to time constrain in 4.16 they may be done or may not, So lets keep this under a generic name.

Reply:
then let's call it `MANAGED_APP_NS_ANNOTATION`...

Reply:
They are going to add this annotation for discovered apps too, they will make it a list of NS

Reply:
ok